exp/api/v1beta1/azuremanagedmachinepool_types.go

```diff
@@ -205,6 +486,10 @@ type AzureManagedMachinePoolSpec struct {
 	// +kubebuilder:validation:Enum=OS;Temporary
 	// +optional
 	KubeletDiskType *KubeletDiskType `json:"kubeletDiskType,omitempty"`
+
+	// LinuxOSConfig specifies the custom linux OS settings and configurations
```

Comment:
Probably "LinuxOSConfig specifies the custom **L**inux OS settings and configurations"?

Reply:
Updated!